models/users.js

```diff
@@ -14,7 +14,7 @@ function createUser(userName, password, firstName, lastName) {
     newUser.session = null;
     newUser.role = 'user';
 
-    return newUser;
+    setUsers(newUser);
```

Comment:
Will this overwrite the existing users?

Reply:
sorry, didn't see that comming. I thought I wrote everything.  Fixed